raw_data/preprocessed/africa/uganda_english.txt

```diff
@@ -0,0 +1,364 @@
+
+Local Environment Committee. 
+(17) [1] A Local Government system shall on the advise of the District Environment Committee, appoint Local Environment Committees. [2] When appointed, the functions of the Local Environment Committee shall include the following [a] to prepare a Local Environment Work Plan which shall be consistent with the National Environment Action Plan and the District Environment Action Plan; [b] to carry out public environmental education campaigns; [c] to mobilize the people within its local jurisdiction to conserve natural resources through self-help; [d] to mobilize the people within its local jurisdiction to restore degraded environmental resources through self-help; [e] to mobilize the people within its local jurisdiction to improve their natural environment through voluntary self-help; [f] to monitor all activities within its local jurisdiction to ensure that such activities do not have any significant impact on the environment; [g] to report any events or activities which have or are likely to have significant impacts on the environment to the or to the appropriate Resistance Committee, Council or such other person as the District Resistance Council may direct; [h] to carry out such other duties as may be prescribed by the District Resistance Council or Urban Council in consultation with the Authority. 
+
+PART IV - ENVIRONMENTAL PLANNING
+
+Environmental planning at national level. 
+(18) [1] The Authority shall prepare a National Environment Action Plan to be reviewed after every five years or such other lesser period as may be considered Environment planning at national necessary by the Authority. [2] The Plan shall: [a] cover all matters affecting the environment of Uganda and shall contain guidelines for the management and protection of the environment and natural resources as well as the strategies for preventing, controlling or mitigating any deleterious effects; [b] take into account district plans as provided for under section 19; [c] be binding upon all persons and all Government departments, agencies and organs; [d] without prejudice to subsection [1], be reviewed and modified from time to time to take into account emerging knowledge and realities; [e] be in such a form and shall contain such other matters as may be prescribed; [f] be subject to approval by the Cabinet and after approval, shall belaid before Parliament; [g] be disseminated to the public. 
```

Comment:
Please remove `Environment planning at national` from the first sub-section. Also, please replace `belaid` with `be laid`.